src/gui/processing/qgsprocessingtoolboxmodel.h

```diff
@@ -303,6 +355,7 @@ class GUI_EXPORT QgsProcessingToolboxModel : public QAbstractItemModel
       AlgorithmShortDescription SIP_MONKEYPATCH_COMPAT_NAME( RoleAlgorithmShortDescription ), //!< Short algorithm description, for algorithm nodes
       AlgorithmTags SIP_MONKEYPATCH_COMPAT_NAME( RoleAlgorithmTags ),                         //!< List of algorithm tags, for algorithm nodes
       ProviderFlags SIP_MONKEYPATCH_COMPAT_NAME( RoleProviderFlags ),                         //!< Returns the node's provider flags
+      ParameterTypeId SIP_MONKEYPATCH_COMPAT_NAME( RoleParameterTypeId ),                     //!< Untranslated parameter type unique identifier for parameter nodes
```

Comment:
```suggestion
      ParameterTypeId,                     //!< Untranslated parameter type unique identifier for parameter nodes. \since QGIS 3.44
```